CollAction/Frontend/app/global/MobileNavDraw.tsx

```diff
@@ -12,23 +15,23 @@ interface IMobileNavDrawState {
 function AccountManageNav(props) {
   if (props.isLoggedIn) {
     return <li><a href="/manage">Account</a></li>;
-  }else {
+  } else {
```

Comment:
Shall we use a newline in these "if else" constructs?

eg. 
```
if (true) {
  ...
}
else {
  ...
}
```

Reply:
fixed in 81a0d8e